nix/status-go/build-mobile-status-go.nix

```diff
@@ -31,10 +31,11 @@ let
       in with targetConfig; ''
       mkdir ${NIX_GOWORKDIR}
 
-      GOPATH=${gomobile.dev}:$GOPATH \
-      PATH=${makeBinPath [ gomobile.bin ]}:$PATH \
-      NIX_GOWORKDIR=${NIX_GOWORKDIR} \
-      ${concatStringsSep " " envVars} \
+      export GO111MODULE=off
```

Comment:
Was there an error that you saw that caused us to need to export the variables?

Reply:
```
Building mobile library for android

type-checking package "github.com/status-im/status-go/mobile" failed (/build/go/src/github.com/status-im/status-go/mobile/multiaccount.go:4:2: could not import encoding/json (go/build: importGo encoding/json: exit status 2
go: cannot find GOROOT directory: /nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-go-1.13.4/share/go
```